homeassistant/components/squeezebox/services.yaml

```diff
@@ -30,3 +30,82 @@ call_query:
       advanced: true
       selector:
         object:
+search:
+  target:
+    entity:
+      integration: squeezebox
+      domain: media_player
+  fields:
+    command:
+      required: true
+      selector:
+        select:
+          translation_key: "search_command_selector"
+          multiple: false
+          options:
+            - albums
+            - artists
+            - tracks
+            - playlists
+            - genres
+            - favorites
+            - players
+    return_items:
+      required: true
+      selector:
+        number:
+          min: 1
+          max: 10000
+          mode: box
+    search_string:
+      required: false
+      example: Revolver
+      selector:
+        text:
+    tags:
+      required: false
+      selector:
+        text:
+play:
+  target:
+    entity:
+      integration: squeezebox
+      domain: media_player
+  fields:
+    command:
+      required: true
+      selector:
+        select:
+          translation_key: "play_command_selector"
+          multiple: false
+          options:
+            - album
+            - artist
+            - track
+            - playlist
+            - genre
+            - favorite
```

Comment:
Please use yaml anchors to avoid the repetitions

Reply:
not sure what you mean here.  There aren't repetitions.  Search is albums, artists (plural), whereas play is album, artist (singular).  You search albums, but you play an album (at least that's the way the API works.)